pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/statics/DBConstants.java

```diff
@@ -542,7 +541,7 @@ public class DBConstants {
     public static final String TRACKING = "tracking";
 
     //EVENTS
-    public static final String TISSUE_RECEIVED_EVENT = "TISSUE_RECEIVED";
+    public static final String REQUIRED_SAMPLES_RECEIVED_EVENT = "REQUIRED_SAMPLES_RECEIVED";
```

Comment:
renaming the event because of the changed logic of when DSM is going to trigger DSS